dulwich/tests/test_porcelain.py

```diff
@@ -1820,6 +1820,11 @@ def test_status_crlf_mismatch(self):
         with open(file_path, "wb") as f:
             f.write(b"line1\r\nline2")
 
+        if sys.platform == "win32":
+            c = self.repo.get_config()
+            c.set("core", "autocrlf", "false")
+            c.write_to_path()
```

Comment:
On windows, this case is only a mismatch if autocrlf is false (if it's true/auto, then `\r\n` in the working dir copy is expected and would be normalized to `\n` in the status call)